mint-auditor/src/gnosis/config.rs

```diff
@@ -88,6 +93,13 @@ impl GnosisSafeConfig {
             .find(|safe| safe.safe_addr == *safe_addr)
             .cloned()
     }
+    /// Get an [AuditedToken] by its [TokenId].
+    pub fn get_tokens_by_token_id(&self, token_id: &TokenId) -> Vec<&AuditedToken> {
```

Comment:
I think this is no longer used and can be removed.

Reply:
Agreed.